azimuth/config.py

```diff
@@ -519,6 +532,14 @@ class AzimuthConfigHistory(AzimuthBaseSettings):
     created_on: str = Field(default_factory=lambda: str(datetime.now(timezone.utc)))
 
 
+class AzimuthConfigHistoryWithHash(AzimuthConfigHistory):
+    hash: str = ""
+
+    @root_validator(skip_on_failure=True)
+    def _set_hash(cls, values):
+        return {**values, "hash": md5_hash(values["config"].dict(by_alias=False))}
```

Comment:
We use `by_alias=True` in other hash, was there a reason here to use `False`?

Reply:
Oh yeah, good question. No reason. As long as it uses the same hash for all the items, it serves the purpose of comparing them.

Anyway, I think `by_alias=True` has no effect on the config since it doesn't have any aliases. For example, our `AliasModel` has an alias generator to make all the field names camelCase for the JSON API. However, `AzimuthConfig` doesn't inherit from `AliasModel`.

Or am I missing something?

If not, I'll remove it.
```suggestion
        return {**values, "hash": md5_hash(values["config"].dict())}
```

Reply:
Yeah, I was wondering too if it was making a difference. Perhaps we should remove it in the 2 other places where we generate the hash then, for consistency.

Reply:
I removed them in 19c6780 and I restarted Azimuth, which read from cache as expected, so the hashes seem identical.

Reply:
Good test! Thanks!

Reply:
I committed 3d3e552 too, to clean things up while it's fresh.

Reply:
Quick check if it makes sense to you?